CODE-OF-CONDUCT.md

```diff
@@ -1,3 +1 @@
-## The Youki Project Community Code of Conduct
-
-The Youki Project follows the [Containers Community Code of Conduct](https://github.com/containers/common/blob/main/CODE-OF-CONDUCT.md).
+Move to containers.github.io/youki/community/goversance.html#code-of-conduct
```

Comment:
I'd say we keep the separate file, it's common practice for CNCF projects: https://github.com/cncf/project-template/blob/main/CODE_OF_CONDUCT.md